integrations/aws/utils/resources.py

```diff
@@ -127,7 +127,7 @@ async def resync_custom_kind(
     next_token = None
     if not describe_method_params:
         describe_method_params = {}
-    while await update_available_access_credentials():
```

Comment:
?

Reply:
While it enhances the frequency at which we check for the token's existence, when it detects that the token isn't available, it causes a thundering herd when the `resync_custom_kind` function is executed concurrently.